bears/r/FormatRBear.py

```diff
@@ -29,7 +29,7 @@ def create_arguments(filename, file, config_file,
                          r_braces_on_next_line: bool=False,
                          r_use_arrows: bool=False,
                          tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
-                         r_max_expression_length: int=20):
+                         r_max_expression_length: int=0):
```

Comment:
I think this suffices:
```
FormatRBear: Disable 'r_max_expression_length'
```
btw: for the commit body you are allowed to use 73 chars per line (or 72, ahh just use 72 :3). Makes stuff better readable :+1: